test/bitfields_packed.ll

```diff
@@ -0,0 +1,125 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv --spirv-debug-info-version=ocl-100 %t.bc
+; RUN: llvm-spirv --spirv-ext=+SPV_KHR_non_semantic_info --spirv-debug-info-version=nonsemantic-shader-100 %t.bc
```

Comment:
```suggestion
; RUN: llvm-spirv --spirv-debug-info-version=nonsemantic-shader-100 %t.bc
```

Reply:
I added this because local testing failed without this.

Reply:
Okay, I'll take a look at this later